### ISSUE TITLE
Import of .csv fails

### ISSUE THREAD

**Issue by mtomasini** [Akodiat/volcanicClouds#2]
The import of .txt evaluation files works as expected (with wrong results), however now the import of .csv files generated through Matlab fails. The reason seems to be that the function to automatically detect which volcano is being looked only works for evaluation logs but is called for any file. I will try to find a work around.

**Comment by Akodiat** [comment]
Sorry for late reply here, but as stated in the Readme: all generated .csv files must be included, together with the Evaluation logs. Do not include the summary file.

For example:
- [EvaluationLog_D2J2819_2024.01.28.txt](https://github.com/Akodiat/volcanicClouds/blob/main/matlab/tomoInverse/sabancaya/EvaluationLog_D2J2819_2024.01.28.txt)
- [EvaluationLog_D2J2833_2024.01.28.txt](https://github.com/Akodiat/volcanicClouds/blob/main/matlab/tomoInverse/sabancaya/EvaluationLog_D2J2833_2024.01.28.txt)	
- [tomography_D2J2819_20240128_1213_D2J2833_20240128_1200.csv](https://github.com/Akodiat/volcanicClouds/blob/main/matlab/tomoInverse/sabancaya/tomography_D2J2819_20240128_1213_D2J2833_20240128_1200.csv)
- [tomography_D2J2819_20240128_1213_D2J2833_20240128_1215.csv](https://github.com/Akodiat/volcanicClouds/blob/main/matlab/tomoInverse/sabancaya/tomography_D2J2819_20240128_1213_D2J2833_20240128_1215.csv)
- [tomography_D2J2819_20240128_1230_D2J2833_20240128_1215.csv](https://github.com/Akodiat/volcanicClouds/blob/main/matlab/tomoInverse/sabancaya/tomography_D2J2819_20240128_1230_D2J2833_20240128_1215.csv)

**Comment by mtomasini** [comment]
Yeah, I did not realize that it required a very specific set of files. I had tried with the summary and it did not work so I assumed it was broken ^^' my bad!
